tools/iothub-explorer/iothub-explorer.js

```diff
@@ -321,6 +323,9 @@ function serviceError(err) {
     }
     console.log(colorsTmpl('\n{bold}{red}Error{/red}{/bold}' + message));
   }
+  else {
+    console.log(colorsTmpl('\n{bold}{red}Raw error message: ' + Util.inspect(err) + '{/red}{/bold}'));
```

Comment:
To be consistent with the non-raw error message, I would only show "Raw error message" in red, and maybe the rest using the regular console font.